### DISCUSSION TITLE
How do I center a div

### DISCUSSION THREAD

**Discussion by jose-izarra** [jose-izarra/attendance_system#5] [Q&A] [answered]
Please help me

**Comment by haxybaxy** [comment, marked as answer]
Method 1: Using Flexbox (Center Both Horizontally and Vertically)
Apply CSS with display: flex; justify-content: center; align-items: center;
Optionally set height: 100vh; for full viewport height.

Method 2: Using Auto Margins (Center Horizontally)
Apply CSS with margin: 0 auto; width: 50%;
Adjust width as needed.

Method 3: Using Absolute Positioning (Center Horizontally and Vertically)
Apply CSS with position: absolute; top: 50%; left: 50%; transform: translate(-50%, -50%);
